pkg/controller/operator/common/defaults.go

```diff
@@ -190,6 +190,8 @@ var (
 	DefaultKubernetesVersioning = operatorv1alpha1.KubermaticVersioningConfiguration{
 		Default: semver.MustParse("v1.19.3"),
 		Versions: []*semver.Version{
+			// Kubernetes 1.16
+			semver.MustParse("v1.16.13"),
```

Comment:
Mmm, I was thinking that putting back a `v1.16` version just to put the auto-upgrade from `1.15` to `1.16` does not seem to be a good option. This version will be visible to users. I think that for the long term we should rework the auto-upgrade mechanism to cover the case where the destination minor version is not supported, but for the moment I would just remove this and the auto-upgrade `1.15` -> `1.16`, customers that still have 1.15 clusters will need to override the default settings to be able to upgrade. WDYT?

Reply:
That's fine

If that cluster version is Out Of Life, no point for the customer using it

Reply:
Ok, you can remove this version then

Reply:
I've removed those and updating the docs for the next push